miden-lib/src/transaction/mod.rs

```diff
@@ -161,6 +160,50 @@ impl TransactionKernel {
             .expect("Invalid stack input")
     }
 
+    /// Extends the advice inputs with account data and Merkle proofs.
+    pub fn extend_advice_inputs_for_account(
```

Comment:
nit: It would b e nice to explain that `merkle_path` should be a path that corresponds to the note root for the account tree of the transaction block header, since it has a generic name for a parameter.